tutorials/2d/2d_meshes.rst

```diff
@@ -1,39 +1,37 @@
-. _doc_2d_meshes:
+.. _doc_2d_meshes:
 
 2D meshes
 =========
 
 Introduction
 ------------
 
-In 3D, meshes are the key type of data used to display the world. In 2D, they are rarer as mostly images are used.
-Godot's 2D engine is a pure two dimensional engine, so it can't really display 3D meshes directly (this is usually done
+In 3D, meshes are used to display the world. In 2D, they are rare as images are used more often.
+Godot's 2D engine is a pure two dimensional engine, so it can't really display 3D meshes directly (although it can be done
 via ``Viewport`` and ``ViewportTexture``).
 
 .. seealso:: If you are interested in displaying 3D meshes on a 2D viewport, see the :ref:`doc_viewport_as_texture` tutorial.
 
-
-From Godot 3.1 onward, support is added for 2D meshes. These are meshes that contain two dimensional geometry (Z can be omitted or ignored) instead of 3D.
-You can experiment creating them yourself using ``SurfaceTool`` from code and setting them in a ``Mesh2D`` node.
+2D meshes are meshes that contain two dimensional geometry (Z can be omitted or ignored) instead of 3D.
+You can experiment creating them yourself using ``SurfaceTool`` from code and displaying them in a ``MeshInstance2D`` node.
```

Comment:
I think what was meant is that the output of `SurfaceTool` should be saved in a `Mesh2D` (and then yes displayed in `MeshInstance2D`).

Reply:
Nevermind, `Mesh2D` doesn't exist... @reduz doesn't know the name of his own classes :P